lib/utils/exception_handler.dart

```diff
@@ -93,7 +99,7 @@ class ExceptionHandler {
       return;
     }
 
-    _saveException(
+    await _saveException(
```

Comment:
we don't want to await saving the exception, we can still show the user the popup until it gets saved